flink-formats/flink-avro-glue-schema-registry/src/main/java/org/apache/flink/formats/avro/glue/schema/registry/GlueSchemaRegistryInputStreamDeserializer.java

```diff
@@ -36,23 +36,24 @@
  * from it and remove schema registry information in the input stream.
  */
 public class GlueSchemaRegistryInputStreamDeserializer {
```

Comment:
This class does not have an annotation to indicate its stability. Please [annotate](https://github.com/apache/flink/tree/master/flink-annotations/src/main/java/org/apache/flink/annotation) using `PublicEvolving/Public/Internal`. This change looks backwards incompatible, so I would favour `PublicEvolving` if this is a public class

Reply:
Please make sure all classes in the src code are annotated appropriately

Reply:
Updated the annotation to `PublicEvolving` in all classes.